src/js/Toolbar/L.PM.Toolbar.js

```diff
@@ -220,6 +222,22 @@ const Toolbar = L.Class.extend({
       actions: ['cancel'],
     };
 
+    const drawCircleMarkerButton = {
+      title: 'Draw Circle Marker',
```

Comment:
Oh another thing, we have recently released a way to internationalize the plugin. Please check the other buttons and add these English titles and tests to the translation file and include it via like you can see in the other buttons. That way, the community can translate the texts into other languages.